apps/explorer/src/ui/DisclosureBox.tsx

```diff
@@ -44,9 +44,9 @@ export function DisclosureBox({
                             as="div"
                             className="flex cursor-pointer flex-nowrap items-center px-5 py-3.75"
                         >
-                            <div className="flex flex-1 gap-1 text-body font-semibold text-gray-90">
+                            <div className="flex w-11/12 flex-1 gap-1 text-body font-semibold text-gray-90">
```

Comment:
Setting the width help to truncate the extra long test and reduce the spacing so open/close remains visible, regardless to the text size

Reply:
The intention here is to make sure the Disclosurebox  show hide icon remains visible even with long text both on mobile and desktop 

with w-full
<img width="367" alt="Screenshot 2023-04-14 at 11 31 35 AM" src="https://user-images.githubusercontent.com/126525197/232088738-4c642cbf-4ca8-4812-85e3-289121037575.png">

w-11/12
<img width="366" alt="Screenshot 2023-04-14 at 11 30 58 AM" src="https://user-images.githubusercontent.com/126525197/232088823-b8456f29-3e39-4455-b96d-fe082d9c3241.png">